lab#1/task#2/scripts/db.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+dbfile=../data/users.db
+inverse=$2
+
+function createFile () {
+	if [ ! -f $dbfile ]; then
+		touch $dbfile;
+	fi
+}
+
+function listItems () {
+	if [ -z $inverse ]; then
+		nl $dbfile
+	else
+		nl $dbfile | sort -nr
+	fi
+	
+}
+
+function add () {
+	createFile;
+	read -p "Enter user name: " username
+	while [ -z $username ]; # Q: How I can pass string with space without double quotes?
```

Comment:
I am not sure what the issue you've faced passing strings with spaces. I tried simple scenario, everything works as expected

```
read -p "Enter user name: " username
echo "$username"

// prints string with spaces
```